tests/test_ghosting.py

```diff
@@ -40,16 +44,18 @@ def test_calculate_ghost_intensity(self):
 
         with pytest.raises(Exception):
             hazen_ghosting.calculate_ghost_intensity(ghost=np.asarray([-10]),
-                                                     phantom=np.asarray([-100]),
-                                                     noise=np.asarray([-5]))
+             phantom=np.asarray([-100]),
+         noise=np.asarray([-5]))
 
         assert 5.0 == hazen_ghosting.calculate_ghost_intensity(ghost=np.asarray([10]),
-                                                               phantom=np.asarray([100]),
-                                                               noise=np.asarray([5]))
+                                               phantom=np.asarray([100]),
+                                        noise=np.asarray([5]))
+
+            # assert -5.0 == hazen_ghosting.calculate_ghost_intensity(ghost=np.asarray([5]),
+            #                    phantom=np.asarray([100]),
+        #                   noise=np.asarray([10]))
```

Comment:
I think you can remove this – assume your abs() correction has made this redundant.

Reply:
yes I think so too. Will remove then.